src/cmd/alias_test.go

```diff
@@ -0,0 +1,29 @@
+package cmd
+
+import "testing"
+
+func TestAliasPreRunE(t *testing.T) {
+	cases := []struct {
+		name   string
+		args   []string
+		expErr bool
+	}{
+		{"TrueArgument", []string{"true"}, false},
+		{"FalseArgument", []string{"false"}, false},
+		{"NoArguments", []string{}, true},
+		{"MutipleArguments", []string{"true", "false"}, true},
+		{"InvalidArgument", []string{"f"}, true},
+	}
+
+	for _, tc := range cases {
+		t.Run(tc.name, func(t *testing.T) {
+			err := aliasCommand.PreRunE(nil, tc.args)
+			if tc.expErr && err == nil {
+				t.Error("expected PreRunE to return an error")
+			}
+			if !tc.expErr && err != nil {
+				t.Errorf("expeced err to be nil; got: %v", err)
+			}
+		})
+	}
+}
```

Comment:
We are using ginkgo / gomega for unit testing. 

Could you please convert this to that framework or move it to a feature test that lives in `features/git-town-alias/alias.feature`. It could have the content

```gherkin
  Scenario: too few arguments
    When I run `git-town alias`
    Then it prints the error "Too few arguments"
    And it prints the error:
      """
      Usage:
        git-town alias (true | false) [flags]
      """
```